bots/discord/game-server-watcher/egg-game-server-watcher.json

```diff
@@ -23,7 +23,7 @@
     "scripts": {
         "installation": {
             "script": "#!\/bin\/bash\r\napt-get update && apt-get install ca-certificates git -yq --no-install-suggests --no-install-recommends --allow-downgrades --allow-remove-essential --allow-change-held-packages\r\ncd \/mnt\/server\r\ngit config --global --add safe.directory \/mnt\/server\r\ngit clone https:\/\/github.com\/a-sync\/game-server-watcher.git . || git pull\r\nnpm install && npm run build\r\nnpm prune --omit=dev",
-            "container": "node:20-bookworm-slim",
+            "container": "node:20-bullseye-slim",
```

Comment:
why switch back? Bookworm is stable. Bullseye oldstable

Reply:
stay on bookworm

Reply:
yolks:nodejs_20 uses the same image (node20:bullseye-slim) as base image. this way they can share image layers. (and not have to download an additional image just for install)

was also thinking about just using the yolk image as installer since it has certs and git installed, but none of the eggs is checked did this, so there might be a reason for it ¯\_(ツ)_/¯

Reply:
No need to use our yolks as installer if it is not a steamcmd egg

Reply:
no reference to steamcmd in [yolks:nodejs_20](https://github.com/parkervcp/yolks/blob/master/nodejs/20/Dockerfile)

will have to test it, but the yolk used for running seems perfectly setup to install what's needed.
it could remove the apt install steps from my install script and possibly fix the git ownership issue currently worked around with `safe.directory` config.

using the same image for install would save up on resources and setup time. unless i'm missing something which is very much possible then please tell me! :D
(documentation on pterodactyl egg workings is very scarce, i wish i read the contribution guide way sooner!)

Reply:
okay it didn't work 🤡
(forgot about the container user)
```
container@pterodactyl~ Starting installation process, this could take a few minutes...
bash: /mnt/install/install.sh: Permission denied
```

anyway i think bullseye is a more economic choice as long as its the base image for the yolk used. (shared layers)